lib/classes/http.rb

```diff
@@ -2,7 +2,67 @@ module Quickbase
   class HTTP
     include HTTParty
     attr_accessor :qb_params
-  
+    module Quickbase
+      class HTTP
+        include HTTParty
+        attr_accessor :qb_params
+
+        def initialize(config)
+          self.class.base_uri "https://#{config[:org]}.quickbase.com"
+          instance_variable_set "@qb_params", {:dbid => "main"}
+          http_proxy = config[:http_proxy] || ENV['http_proxy']
+          setup_proxy(http_proxy) if http_proxy
+
+          response = post("API_Authenticate", Quickbase::Helper.hash_to_xml(config))
```

Comment:
can this be called outsize the initialize call?  Don't typically like to have stuff that can fail in the constructor.